views/js/runner/provider/qti.js

```diff
@@ -400,12 +394,29 @@ define([
                 this.getProbeOverseer().start();
             }
 
-            //load data and current context in parrallel at initialization
+            //load data and current context in parallel at initialization
             return this.getProxy().init()
                        .then(function(results){
+                            var communicator = self.getProxy().getCommunicator();
+
                             self.setTestData(results.testData);
                             self.setTestContext(results.testContext);
                             self.setTestMap(results.testMap);
+
+                            // install the communication channel if enabled
+                            if (communicator) {
```

Comment:
This will be difficult to put this part in the proxy, as this code needs to access to the runner instance.